<?xml version='1.0' encoding='UTF-8'?>
<root>
 <tabbed_widget parent="main_window" name="Main Window">
  <Tab tab_name="cal_data" containers="1">
   <Container>
    <DockSplitter orientation="-" count="2" sizes="0.816193;0.183807">
     <DockSplitter orientation="|" count="2" sizes="0.500695;0.499305">
      <DockSplitter orientation="-" count="4" sizes="0.250336;0.248991;0.251682;0.248991">
       <DockArea name="...">
        <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
         <range left="3.425500" bottom="-0.100000" top="0.100000" right="12.425500"/>
         <limitY/>
         <curve name="/mhe/jnt_calib_status/K_d0_cal.0" color="#9467bd"/>
         <curve name="/mhe/jnt_calib_status/K_d0_cal.1" color="#17becf"/>
         <curve name="/mhe/jnt_calib_status/K_d0_cal.2" color="#bcbd22"/>
         <curve name="/mhe/jnt_calib_status/K_d0_cal.3" color="#1f77b4"/>
         <curve name="/mhe/jnt_calib_status/K_d0_cal.4" color="#d62728"/>
        </plot>
       </DockArea>
       <DockArea name="...">
        <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
         <range left="3.425500" bottom="-3.247239" top="0.551743" right="12.425500"/>
         <limitY/>
         <curve name="/mhe/jnt_calib_status/K_d1_cal.0" color="#1ac938"/>
         <curve name="/mhe/jnt_calib_status/K_d1_cal.1" color="#ff7f0e"/>
         <curve name="/mhe/jnt_calib_status/K_d1_cal.2" color="#f14cc1"/>
         <curve name="/mhe/jnt_calib_status/K_d1_cal.3" color="#9467bd"/>
         <curve name="/mhe/jnt_calib_status/K_d1_cal.4" color="#17becf"/>
        </plot>
       </DockArea>
       <DockArea name="...">
        <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
         <range left="3.425500" bottom="0.040000" top="0.240000" right="12.425500"/>
         <limitY/>
         <curve name="/mhe/jnt_calib_status/K_t_cal.0" color="#bcbd22"/>
         <curve name="/mhe/jnt_calib_status/K_t_cal.1" color="#1f77b4"/>
         <curve name="/mhe/jnt_calib_status/K_t_cal.2" color="#d62728"/>
         <curve name="/mhe/jnt_calib_status/K_t_cal.3" color="#1ac938"/>
         <curve name="/mhe/jnt_calib_status/K_t_cal.4" color="#ff7f0e"/>
        </plot>
       </DockArea>
       <DockArea name="...">
        <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
         <range left="3.425500" bottom="0.000147" top="0.000330" right="12.425500"/>
         <limitY/>
         <curve name="/mhe/jnt_calib_status/rotor_MoI_cal.0" color="#f14cc1"/>
         <curve name="/mhe/jnt_calib_status/rotor_MoI_cal.1" color="#9467bd"/>
         <curve name="/mhe/jnt_calib_status/rotor_MoI_cal.2" color="#17becf"/>
         <curve name="/mhe/jnt_calib_status/rotor_MoI_cal.3" color="#bcbd22"/>
         <curve name="/mhe/jnt_calib_status/rotor_MoI_cal.4" color="#1f77b4"/>
        </plot>
       </DockArea>
      </DockSplitter>
      <DockSplitter orientation="-" count="4" sizes="0.250336;0.250336;0.248991;0.250336">
       <DockArea name="...">
        <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
         <range left="3.425500" bottom="-0.100000" top="0.100000" right="12.425500"/>
         <limitY/>
         <curve name="/mhe/jnt_calib_status/alpha_d0.0" color="#d62728"/>
         <curve name="/mhe/jnt_calib_status/alpha_d0.1" color="#1ac938"/>
         <curve name="/mhe/jnt_calib_status/alpha_d0.2" color="#ff7f0e"/>
         <curve name="/mhe/jnt_calib_status/alpha_d0.3" color="#f14cc1"/>
         <curve name="/mhe/jnt_calib_status/alpha_d0.4" color="#9467bd"/>
        </plot>
       </DockArea>
       <DockArea name="...">
        <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
         <range left="3.425500" bottom="-0.000001" top="0.000001" right="12.425500"/>
         <limitY/>
         <curve name="/mhe/jnt_calib_status/alpha_d1.0" color="#17becf"/>
         <curve name="/mhe/jnt_calib_status/alpha_d1.1" color="#bcbd22"/>
         <curve name="/mhe/jnt_calib_status/alpha_d1.2" color="#1f77b4"/>
         <curve name="/mhe/jnt_calib_status/alpha_d1.3" color="#d62728"/>
         <curve name="/mhe/jnt_calib_status/alpha_d1.4" color="#1ac938"/>
        </plot>
       </DockArea>
       <DockArea name="...">
        <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
         <range left="3.425500" bottom="-0.007925" top="0.007930" right="12.425500"/>
         <limitY/>
         <curve name="/mhe/jnt_calib_status/alpha_inertial.0" color="#ff7f0e"/>
         <curve name="/mhe/jnt_calib_status/alpha_inertial.1" color="#f14cc1"/>
         <curve name="/mhe/jnt_calib_status/alpha_inertial.2" color="#9467bd"/>
         <curve name="/mhe/jnt_calib_status/alpha_inertial.3" color="#17becf"/>
         <curve name="/mhe/jnt_calib_status/alpha_inertial.4" color="#bcbd22"/>
        </plot>
       </DockArea>
       <DockArea name="...">
        <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
         <range left="3.425500" bottom="-0.100000" top="0.100000" right="12.425500"/>
         <limitY/>
         <curve name="/mhe/jnt_calib_status/alpha_kt.0" color="#1f77b4"/>
         <curve name="/mhe/jnt_calib_status/alpha_kt.1" color="#d62728"/>
         <curve name="/mhe/jnt_calib_status/alpha_kt.2" color="#1ac938"/>
         <curve name="/mhe/jnt_calib_status/alpha_kt.3" color="#ff7f0e"/>
         <curve name="/mhe/jnt_calib_status/alpha_kt.4" color="#f14cc1"/>
        </plot>
       </DockArea>
      </DockSplitter>
     </DockSplitter>
     <DockArea name="...">
      <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
       <range left="3.425500" bottom="-0.025000" top="1.025000" right="12.425500"/>
       <limitY/>
       <curve name="/mhe/jnt_calib_status/sol_millis.0" color="#9467bd"/>
       <curve name="/mhe/jnt_calib_status/sol_millis.1" color="#17becf"/>
       <curve name="/mhe/jnt_calib_status/sol_millis.2" color="#bcbd22"/>
       <curve name="/mhe/jnt_calib_status/sol_millis.3" color="#1f77b4"/>
       <curve name="/mhe/jnt_calib_status/sol_millis.4" color="#d62728"/>
      </plot>
     </DockArea>
    </DockSplitter>
   </Container>
  </Tab>
  <Tab tab_name="cal_state" containers="1">
   <Container>
    <DockSplitter orientation="-" count="4" sizes="0.25;0.251096;0.248904;0.25">
     <DockArea name="...">
      <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
       <range left="3.425500" bottom="-0.000105" top="0.000146" right="12.425500"/>
       <limitY/>
       <curve name="/mhe/jnt_calib_status/q_dot.0" color="#1ac938"/>
       <curve name="/mhe/jnt_calib_status/q_dot.1" color="#ff7f0e"/>
       <curve name="/mhe/jnt_calib_status/q_dot.2" color="#f14cc1"/>
       <curve name="/mhe/jnt_calib_status/q_dot.3" color="#17becf"/>
       <curve name="/mhe/jnt_calib_status/q_dot.4" color="#9467bd"/>
      </plot>
     </DockArea>
     <DockArea name="...">
      <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
       <range left="3.425500" bottom="-0.000066" top="0.000066" right="12.425500"/>
       <limitY/>
       <curve name="/mhe/jnt_calib_status/q_ddot.0" color="#bcbd22"/>
       <curve name="/mhe/jnt_calib_status/q_ddot.1" color="#1f77b4"/>
       <curve name="/mhe/jnt_calib_status/q_ddot.2" color="#d62728"/>
       <curve name="/mhe/jnt_calib_status/q_ddot.3" color="#1ac938"/>
       <curve name="/mhe/jnt_calib_status/q_ddot.4" color="#ff7f0e"/>
      </plot>
     </DockArea>
     <DockArea name="...">
      <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
       <range left="3.425500" bottom="-0.100000" top="0.100000" right="12.425500"/>
       <limitY/>
       <curve name="/mhe/jnt_calib_status/iq.0" color="#f14cc1"/>
       <curve name="/mhe/jnt_calib_status/iq.1" color="#9467bd"/>
       <curve name="/mhe/jnt_calib_status/iq.2" color="#17becf"/>
       <curve name="/mhe/jnt_calib_status/iq.3" color="#bcbd22"/>
       <curve name="/mhe/jnt_calib_status/iq.4" color="#1f77b4"/>
      </plot>
     </DockArea>
     <DockArea name="...">
      <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
       <range left="3.425500" bottom="-59.391490" top="2.048833" right="12.425500"/>
       <limitY/>
       <curve name="/mhe/jnt_calib_status/tau.0" color="#d62728"/>
       <curve name="/mhe/jnt_calib_status/tau.1" color="#1ac938"/>
       <curve name="/mhe/jnt_calib_status/tau.2" color="#ff7f0e"/>
       <curve name="/mhe/jnt_calib_status/tau.3" color="#f14cc1"/>
       <curve name="/mhe/jnt_calib_status/tau.4" color="#9467bd"/>
      </plot>
     </DockArea>
    </DockSplitter>
   </Container>
  </Tab>
  <Tab tab_name="cal_res" containers="1">
   <Container>
    <DockSplitter orientation="-" count="4" sizes="0.25;0.251096;0.248904;0.25">
     <DockSplitter orientation="|" count="3" sizes="0.359777;0.321503;0.31872">
      <DockArea name="...">
       <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
        <range left="3.425500" bottom="0.040000" top="0.240000" right="12.425500"/>
        <limitY/>
        <curve name="/mhe/jnt_calib_status/K_t_cal.0" color="#bcbd22"/>
        <curve name="/mhe/jnt_calib_status/K_t_cal.1" color="#1f77b4"/>
        <curve name="/mhe/jnt_calib_status/K_t_cal.2" color="#d62728"/>
        <curve name="/mhe/jnt_calib_status/K_t_cal.3" color="#1ac938"/>
        <curve name="/mhe/jnt_calib_status/K_t_cal.4" color="#ff7f0e"/>
       </plot>
      </DockArea>
      <DockArea name="...">
       <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
        <range left="3.425500" bottom="0.040000" top="0.240000" right="12.425500"/>
        <limitY/>
        <curve name="/mhe/jnt_calib_status/K_t_ig.0" color="#17becf"/>
        <curve name="/mhe/jnt_calib_status/K_t_ig.1" color="#bcbd22"/>
        <curve name="/mhe/jnt_calib_status/K_t_ig.2" color="#1f77b4"/>
        <curve name="/mhe/jnt_calib_status/K_t_ig.3" color="#d62728"/>
        <curve name="/mhe/jnt_calib_status/K_t_ig.4" color="#1ac938"/>
       </plot>
      </DockArea>
      <DockArea name="...">
       <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
        <range left="3.425500" bottom="0.040000" top="0.240000" right="12.425500"/>
        <limitY/>
        <curve name="/mhe/jnt_calib_status/K_t_nom.0" color="#ff7f0e"/>
        <curve name="/mhe/jnt_calib_status/K_t_nom.1" color="#f14cc1"/>
        <curve name="/mhe/jnt_calib_status/K_t_nom.2" color="#9467bd"/>
        <curve name="/mhe/jnt_calib_status/K_t_nom.3" color="#17becf"/>
        <curve name="/mhe/jnt_calib_status/K_t_nom.4" color="#bcbd22"/>
       </plot>
      </DockArea>
     </DockSplitter>
     <DockSplitter orientation="|" count="3" sizes="0.35769;0.321503;0.320807">
      <DockArea name="...">
       <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
        <range left="3.425500" bottom="-0.100000" top="0.100000" right="12.425500"/>
        <limitY/>
        <curve name="/mhe/jnt_calib_status/K_d0_cal.0" color="#9467bd"/>
        <curve name="/mhe/jnt_calib_status/K_d0_cal.1" color="#17becf"/>
        <curve name="/mhe/jnt_calib_status/K_d0_cal.2" color="#bcbd22"/>
        <curve name="/mhe/jnt_calib_status/K_d0_cal.3" color="#1f77b4"/>
        <curve name="/mhe/jnt_calib_status/K_d0_cal.4" color="#d62728"/>
       </plot>
      </DockArea>
      <DockArea name="...">
       <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
        <range left="3.425500" bottom="-0.100000" top="0.100000" right="12.425500"/>
        <limitY/>
        <curve name="/mhe/jnt_calib_status/K_d0_ig.0" color="#1f77b4"/>
        <curve name="/mhe/jnt_calib_status/K_d0_ig.1" color="#d62728"/>
        <curve name="/mhe/jnt_calib_status/K_d0_ig.2" color="#1ac938"/>
        <curve name="/mhe/jnt_calib_status/K_d0_ig.3" color="#ff7f0e"/>
        <curve name="/mhe/jnt_calib_status/K_d0_ig.4" color="#f14cc1"/>
       </plot>
      </DockArea>
      <DockArea name="...">
       <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
        <range left="3.425500" bottom="-0.100000" top="0.100000" right="12.425500"/>
        <limitY/>
        <curve name="/mhe/jnt_calib_status/K_d0_nom.0" color="#9467bd"/>
        <curve name="/mhe/jnt_calib_status/K_d0_nom.1" color="#17becf"/>
        <curve name="/mhe/jnt_calib_status/K_d0_nom.2" color="#bcbd22"/>
        <curve name="/mhe/jnt_calib_status/K_d0_nom.3" color="#1f77b4"/>
        <curve name="/mhe/jnt_calib_status/K_d0_nom.4" color="#d62728"/>
       </plot>
      </DockArea>
     </DockSplitter>
     <DockSplitter orientation="|" count="3" sizes="0.35769;0.321503;0.320807">
      <DockArea name="...">
       <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
        <range left="3.425500" bottom="-3.247239" top="0.551743" right="12.425500"/>
        <limitY/>
        <curve name="/mhe/jnt_calib_status/K_d1_cal.0" color="#1ac938"/>
        <curve name="/mhe/jnt_calib_status/K_d1_cal.1" color="#ff7f0e"/>
        <curve name="/mhe/jnt_calib_status/K_d1_cal.2" color="#f14cc1"/>
        <curve name="/mhe/jnt_calib_status/K_d1_cal.3" color="#9467bd"/>
        <curve name="/mhe/jnt_calib_status/K_d1_cal.4" color="#17becf"/>
       </plot>
      </DockArea>
      <DockArea name="...">
       <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
        <range left="3.425500" bottom="-0.100000" top="0.100000" right="12.425500"/>
        <limitY/>
        <curve name="/mhe/jnt_calib_status/K_d1_ig.0" color="#1ac938"/>
        <curve name="/mhe/jnt_calib_status/K_d1_ig.1" color="#ff7f0e"/>
        <curve name="/mhe/jnt_calib_status/K_d1_ig.2" color="#f14cc1"/>
        <curve name="/mhe/jnt_calib_status/K_d1_ig.3" color="#9467bd"/>
        <curve name="/mhe/jnt_calib_status/K_d1_ig.4" color="#17becf"/>
       </plot>
      </DockArea>
      <DockArea name="...">
       <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
        <range left="3.425500" bottom="-0.100000" top="0.100000" right="12.425500"/>
        <limitY/>
        <curve name="/mhe/jnt_calib_status/K_d1_nom.0" color="#bcbd22"/>
        <curve name="/mhe/jnt_calib_status/K_d1_nom.1" color="#1f77b4"/>
        <curve name="/mhe/jnt_calib_status/K_d1_nom.2" color="#d62728"/>
        <curve name="/mhe/jnt_calib_status/K_d1_nom.3" color="#1ac938"/>
        <curve name="/mhe/jnt_calib_status/K_d1_nom.4" color="#ff7f0e"/>
       </plot>
      </DockArea>
     </DockSplitter>
     <DockSplitter orientation="|" count="3" sizes="0.35769;0.321503;0.320807">
      <DockArea name="...">
       <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
        <range left="3.425500" bottom="0.000147" top="0.000330" right="12.425500"/>
        <limitY/>
        <curve name="/mhe/jnt_calib_status/rotor_MoI_cal.0" color="#f14cc1"/>
        <curve name="/mhe/jnt_calib_status/rotor_MoI_cal.1" color="#9467bd"/>
        <curve name="/mhe/jnt_calib_status/rotor_MoI_cal.2" color="#17becf"/>
        <curve name="/mhe/jnt_calib_status/rotor_MoI_cal.3" color="#bcbd22"/>
        <curve name="/mhe/jnt_calib_status/rotor_MoI_cal.4" color="#1f77b4"/>
       </plot>
      </DockArea>
      <DockArea name="...">
       <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
        <range left="3.425500" bottom="0.000147" top="0.000330" right="12.425500"/>
        <limitY/>
        <curve name="/mhe/jnt_calib_status/rotor_MoI_ig.0" color="#f14cc1"/>
        <curve name="/mhe/jnt_calib_status/rotor_MoI_ig.1" color="#9467bd"/>
        <curve name="/mhe/jnt_calib_status/rotor_MoI_ig.2" color="#17becf"/>
        <curve name="/mhe/jnt_calib_status/rotor_MoI_ig.3" color="#bcbd22"/>
        <curve name="/mhe/jnt_calib_status/rotor_MoI_ig.4" color="#1f77b4"/>
       </plot>
      </DockArea>
      <DockArea name="...">
       <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
        <range left="3.425500" bottom="0.000147" top="0.000330" right="12.425500"/>
        <limitY/>
        <curve name="/mhe/jnt_calib_status/rotor_MoI_nom.0" color="#d62728"/>
        <curve name="/mhe/jnt_calib_status/rotor_MoI_nom.1" color="#1ac938"/>
        <curve name="/mhe/jnt_calib_status/rotor_MoI_nom.2" color="#ff7f0e"/>
        <curve name="/mhe/jnt_calib_status/rotor_MoI_nom.3" color="#f14cc1"/>
        <curve name="/mhe/jnt_calib_status/rotor_MoI_nom.4" color="#9467bd"/>
       </plot>
      </DockArea>
     </DockSplitter>
    </DockSplitter>
   </Container>
  </Tab>
  <Tab tab_name="cal_reg" containers="1">
   <Container>
    <DockSplitter orientation="-" count="4" sizes="0.25;0.251096;0.248904;0.25">
     <DockArea name="...">
      <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
       <range left="3.425500" bottom="-2499999999999.890625" top="102500000000000.000000" right="12.425500"/>
       <limitY/>
       <curve name="/mhe/jnt_calib_status/lambda_high.0" color="#17becf"/>
       <curve name="/mhe/jnt_calib_status/lambda_des.0" color="#bcbd22"/>
      </plot>
     </DockArea>
     <DockArea name="...">
      <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
       <range left="3.425500" bottom="-24999.998975" top="1024999.999975" right="12.425500"/>
       <limitY/>
       <curve name="/mhe/jnt_calib_status/lambda_high.1" color="#1f77b4"/>
       <curve name="/mhe/jnt_calib_status/lambda_des.1" color="#d62728"/>
      </plot>
     </DockArea>
     <DockArea name="...">
      <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
       <range left="3.425500" bottom="-24999.998975" top="1024999.999975" right="12.425500"/>
       <limitY/>
       <curve name="/traj_replayer/jnt_calib_status/lambda_high.2" color="#9467bd"/>
       <curve name="/traj_replayer/jnt_calib_status/lambda.2" color="#d62728"/>
       <curve name="/mhe/jnt_calib_status/lambda_high.2" color="#1ac938"/>
       <curve name="/mhe/jnt_calib_status/lambda_des.2" color="#ff7f0e"/>
      </plot>
     </DockArea>
     <DockArea name="...">
      <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
       <range left="3.425500" bottom="-2499999999999.890625" top="102500000000000.000000" right="12.425500"/>
       <limitY/>
       <curve name="/mhe/jnt_calib_status/lambda_high.3" color="#f14cc1"/>
       <curve name="/mhe/jnt_calib_status/lambda_des.3" color="#9467bd"/>
      </plot>
     </DockArea>
    </DockSplitter>
   </Container>
  </Tab>
  <Tab tab_name="cal_traj_refs" containers="1">
   <Container>
    <DockSplitter orientation="-" count="3" sizes="0.332968;0.334064;0.332968">
     <DockSplitter orientation="|" count="2" sizes="0.763561;0.236439">
      <DockArea name="...">
       <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
        <range left="3.430500" bottom="-0.089257" top="2.562593" right="12.430000"/>
        <limitY/>
        <curve name="/traj_replayer/calib_traj_status/pos_ref.0" color="#17becf"/>
        <curve name="/traj_replayer/calib_traj_status/pos_ref.1" color="#bcbd22"/>
        <curve name="/traj_replayer/calib_traj_status/pos_ref.2" color="#1f77b4"/>
        <curve name="/traj_replayer/calib_traj_status/pos_ref.3" color="#d62728"/>
        <curve name="/traj_replayer/calib_traj_status/pos_ref.4" color="#1ac938"/>
        <curve name="/traj_replayer/calib_traj_status/pos_ref.5" color="#ff7f0e"/>
        <curve name="/traj_replayer/calib_traj_status/pos_ref.6" color="#f14cc1"/>
        <curve name="/traj_replayer/calib_traj_status/pos_ref.7" color="#9467bd"/>
        <curve name="/traj_replayer/calib_traj_status/pos_ref.8" color="#17becf"/>
        <curve name="/traj_replayer/calib_traj_status/pos_ref.9" color="#bcbd22"/>
        <curve name="/traj_replayer/calib_traj_status/pos_ref.10" color="#1f77b4"/>
        <curve name="/traj_replayer/calib_traj_status/pos_ref.11" color="#d62728"/>
        <curve name="/traj_replayer/calib_traj_status/pos_ref.12" color="#1ac938"/>
       </plot>
      </DockArea>
      <DockArea name="...">
       <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
        <range left="3.430500" bottom="0.900000" top="1.100000" right="12.430000"/>
        <limitY/>
        <curve name="/traj_replayer/calib_traj_status/send_pos" color="#9467bd"/>
       </plot>
      </DockArea>
     </DockSplitter>
     <DockSplitter orientation="|" count="2" sizes="0.76217;0.23783">
      <DockArea name="...">
       <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
        <range left="3.430500" bottom="-0.016857" top="0.000411" right="12.430000"/>
        <limitY/>
        <curve name="/traj_replayer/calib_traj_status/vel_ref.0" color="#ff7f0e"/>
        <curve name="/traj_replayer/calib_traj_status/vel_ref.1" color="#f14cc1"/>
        <curve name="/traj_replayer/calib_traj_status/vel_ref.2" color="#9467bd"/>
        <curve name="/traj_replayer/calib_traj_status/vel_ref.3" color="#17becf"/>
        <curve name="/traj_replayer/calib_traj_status/vel_ref.4" color="#bcbd22"/>
        <curve name="/traj_replayer/calib_traj_status/vel_ref.5" color="#1f77b4"/>
        <curve name="/traj_replayer/calib_traj_status/vel_ref.6" color="#d62728"/>
        <curve name="/traj_replayer/calib_traj_status/vel_ref.7" color="#1ac938"/>
        <curve name="/traj_replayer/calib_traj_status/vel_ref.8" color="#ff7f0e"/>
        <curve name="/traj_replayer/calib_traj_status/vel_ref.9" color="#f14cc1"/>
        <curve name="/traj_replayer/calib_traj_status/vel_ref.10" color="#9467bd"/>
        <curve name="/traj_replayer/calib_traj_status/vel_ref.11" color="#17becf"/>
        <curve name="/traj_replayer/calib_traj_status/vel_ref.12" color="#bcbd22"/>
       </plot>
      </DockArea>
      <DockArea name="...">
       <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
        <range left="3.430500" bottom="0.900000" top="1.100000" right="12.430000"/>
        <limitY/>
        <curve name="/traj_replayer/calib_traj_status/send_vel" color="#17becf"/>
       </plot>
      </DockArea>
     </DockSplitter>
     <DockArea name="...">
      <plot flip_y="false" flip_x="false" mode="TimeSeries" style="Lines">
       <range left="3.430500" bottom="-0.253916" top="0.006193" right="12.430000"/>
       <limitY/>
       <curve name="/traj_replayer/calib_traj_status/acc_ref.0" color="#1f77b4"/>
       <curve name="/traj_replayer/calib_traj_status/acc_ref.1" color="#d62728"/>
       <curve name="/traj_replayer/calib_traj_status/acc_ref.2" color="#1ac938"/>
       <curve name="/traj_replayer/calib_traj_status/acc_ref.3" color="#ff7f0e"/>
       <curve name="/traj_replayer/calib_traj_status/acc_ref.4" color="#f14cc1"/>
       <curve name="/traj_replayer/calib_traj_status/acc_ref.5" color="#9467bd"/>
       <curve name="/traj_replayer/calib_traj_status/acc_ref.6" color="#17becf"/>
       <curve name="/traj_replayer/calib_traj_status/acc_ref.7" color="#bcbd22"/>
       <curve name="/traj_replayer/calib_traj_status/acc_ref.8" color="#1f77b4"/>
       <curve name="/traj_replayer/calib_traj_status/acc_ref.9" color="#d62728"/>
       <curve name="/traj_replayer/calib_traj_status/acc_ref.10" color="#1ac938"/>
       <curve name="/traj_replayer/calib_traj_status/acc_ref.11" color="#ff7f0e"/>
       <curve name="/traj_replayer/calib_traj_status/acc_ref.12" color="#f14cc1"/>
      </plot>
     </DockArea>
    </DockSplitter>
   </Container>
  </Tab>
  <currentTabIndex index="4"/>
 </tabbed_widget>
 <use_relative_time_offset enabled="1"/>
 <!-- - - - - - - - - - - - - - - -->
 <!-- - - - - - - - - - - - - - - -->
 <Plugins>
  <plugin ID="DataLoad CSV">
   <default delimiter="0" time_axis=""/>
  </plugin>
  <plugin ID="DataLoad MCAP"/>
  <plugin ID="DataLoad ROS bags">
   <use_header_stamp value="false"/>
   <discard_large_arrays value="true"/>
   <max_array_size value="100"/>
   <boolean_strings_to_number value="true"/>
   <remove_suffix_from_strings value="true"/>
  </plugin>
  <plugin ID="DataLoad ULog"/>
  <plugin ID="ROS Topic Subscriber">
   <use_header_stamp value="false"/>
   <discard_large_arrays value="true"/>
   <max_array_size value="100"/>
   <boolean_strings_to_number value="true"/>
   <remove_suffix_from_strings value="true"/>
  </plugin>
  <plugin ID="UDP Server"/>
  <plugin ID="WebSocket Server"/>
  <plugin ID="ZMQ Subscriber"/>
  <plugin ID="Fast Fourier Transform"/>
  <plugin ID="Quaternion to RPY"/>
  <plugin ID="Reactive Script Editor">
   <library code="--[[ Helper function to create a series from arrays&#xa;&#xa; new_series: a series previously created with ScatterXY.new(name)&#xa; prefix:     prefix of the timeseries, before the index of the array&#xa; suffix_X:   suffix to complete the name of the series containing the X value. If [nil], use the index of the array.&#xa; suffix_Y:   suffix to complete the name of the series containing the Y value&#xa; timestamp:   usually the tracker_time variable&#xa;              &#xa; Example:&#xa; &#xa; Assuming we have multiple series in the form:&#xa; &#xa;   /trajectory/node.{X}/position/x&#xa;   /trajectory/node.{X}/position/y&#xa;   &#xa; where {N} is the index of the array (integer). We can create a reactive series from the array with:&#xa; &#xa;   new_series = ScatterXY.new(&quot;my_trajectory&quot;) &#xa;   CreateSeriesFromArray( new_series, &quot;/trajectory/node&quot;, &quot;position/x&quot;, &quot;position/y&quot;, tracker_time );&#xa;--]]&#xa;&#xa;function CreateSeriesFromArray( new_series, prefix, suffix_X, suffix_Y, timestamp )&#xa;  &#xa;  --- clear previous values&#xa;  new_series:clear()&#xa;  &#xa;  --- Append points to new_series&#xa;  index = 0&#xa;  while(true) do&#xa;&#xa;    x = index;&#xa;    -- if not nil, get the X coordinate from a series&#xa;    if suffix_X ~= nil then &#xa;      series_x = TimeseriesView.find( string.format( &quot;%s.%d/%s&quot;, prefix, index, suffix_X) )&#xa;      if series_x == nil then break end&#xa;      x = series_x:atTime(timestamp)&#x9; &#xa;    end&#xa;    &#xa;    series_y = TimeseriesView.find( string.format( &quot;%s.%d/%s&quot;, prefix, index, suffix_Y) )&#xa;    if series_y == nil then break end &#xa;    y = series_y:atTime(timestamp)&#xa;    &#xa;    new_series:push_back(x,y)&#xa;    index = index+1&#xa;  end&#xa;end&#xa;&#xa;--[[ Similar to the built-in function GetSeriesNames(), but select only the names with a give prefix. --]]&#xa;&#xa;function GetSeriesNamesByPrefix(prefix)&#xa;  -- GetSeriesNames(9 is a built-in function&#xa;  all_names = GetSeriesNames()&#xa;  filtered_names = {}&#xa;  for i, name in ipairs(all_names)  do&#xa;    -- check the prefix&#xa;    if name:find(prefix, 1, #prefix) then&#xa;      table.insert(filtered_names, name);&#xa;    end&#xa;  end&#xa;  return filtered_names&#xa;end&#xa;&#xa;--[[ Modify an existing series, applying offsets to all their X and Y values&#xa;&#xa; series: an existing timeseries, obtained with TimeseriesView.find(name)&#xa; delta_x: offset to apply to each x value&#xa; delta_y: offset to apply to each y value &#xa;  &#xa;--]]&#xa;&#xa;function ApplyOffsetInPlace(series, delta_x, delta_y)&#xa;  -- use C++ indeces, not Lua indeces&#xa;  for index=0, series:size()-1 do&#xa;    x,y = series:at(index)&#xa;    series:set(index, x + delta_x, y + delta_y)&#xa;  end&#xa;end&#xa;"/>
   <scripts/>
  </plugin>
  <plugin ID="CSV Exporter"/>
  <plugin ID="ROS /rosout Visualization"/>
  <plugin ID="ROS Topic Re-Publisher"/>
 </Plugins>
 <!-- - - - - - - - - - - - - - - -->
 <previouslyLoaded_Datafiles/>
 <previouslyLoaded_Streamer name="ROS Topic Subscriber"/>
 <!-- - - - - - - - - - - - - - - -->
 <customMathEquations/>
 <snippets/>
 <!-- - - - - - - - - - - - - - - -->
</root>

